### ISSUE TITLE
pre-process videos to 25 FPS

### ISSUE THREAD

**Issue by bomri** [deepmind/kinetics-i3d#59]
Hello,
Is it crucial to first encode all videos with 25 FPS, to match the temporal footprint and number of input frames mentioned in the paper?

**Comment by joaoluiscarreira** [comment]
Hi,

I suppose it won't make much of a difference if the videos are 15 FPS or
so, but do not have numbers to back it up.

Best,

Joao

**Comment by bomri** [comment]
Thanks Joao.
I mean in case every video has a different FPS ?

**Comment by joaoluiscarreira** [comment]
Yes, i think it should be alright. In many cases this is similar to having
actions executed slower or faster, as long as it is not too extreme (i
assume slower than 5 fps may be problematic).

Joao

**Comment by aurooj** [comment]
Hi, regarding the same question if the frames were sampled from a video at 3 fps and want to extract I3D features for them, do you think I3d features will be meaningful?

**Comment by joaoluiscarreira** [comment]
I haven't tried it but they'll probably be slightly less accurate than with
higher frame rates. If you don't have computational constraints one
possibility would be to upsample the frame rate by a factor or 3 or so
before inputting to the model.

Joao

**Comment by aurooj** [comment]
I see,. Yeah, I have very long videos to work with. For now, will try feeding them as minimum possible clips i3d model takes and see how reasonable the predictions are. Thanks for your prompt response.
